torch/_dynamo/eval_frame.py

```diff
@@ -1449,6 +1449,13 @@ def patch():
         # Note: we don't support sparsity, data-dependent control, or tracing through backwards
         excluded_opts = {torch.optim.SparseAdam, torch.optim.RAdam, torch.optim.LBFGS}
         for opt in optimizers:
+            # We disable `register_load_state_dict_pre_hook` to allow torch.compile to trace
+            # through the optimizer init without failing. Otherwise we get an irrecoverable graphbreak
+            # saying "torch.* op returned non-Tensor RemovableHandle call_function <class
+            # 'torch.utils.hooks.RemovableHandle'>"
```

Comment:
See failures in https://hud.pytorch.org/pr/106987

Reply:
Sounds like a dynamo bug since graph breaking is fine here. We should open an issue with a small repro for the Dynamo team to look into it.